documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -867,6 +867,11 @@ public class Titles {
 
 [WARNING]
 ====
+When the qualifier is placed in a different library, make sure it has the retention policy Class (`@Retention(RetentionPolicy.CLASS)`). Otherwise the qualifier will not end up in the jar and MapStruct cannot 'see' it during code generation.
```

Comment:
I'd actually just say here and in the javadoc above that `@Retention(RetentionPolicy.CLASS)` should always be used on custom qualifier annotations, just to be on the safe side (they might suddenly be reused in a different module, or someone on the team uses Eclipse and due to the incremental compilation it reads the .class file instead of the .java file from the same project.